dsew_community_profile/delphi_dsew_community_profile/pull.py

```diff
@@ -682,3 +761,37 @@ def std_err(df):
     p = df.val
     n = df.sample_size
     return np.sqrt(p * (1 - p) / n)
+
+def apply_thres_change_date(apply_fn, df, *apply_fn_args):
+    """
+    Apply a function separately to data before and after the test volume change date.
+
+    The test volume change date is when test volume and test positivity started being
+    repported for different reference dates within the same report.
+
+    Parameters
+    ----------
+    apply_fn: function
+        function to apply to data before and after the test volume change date
+    df: pd.DataFrame
+        Columns: val, sample_size, ...
+    apply_fn_args: tuple of lists
+        variable number of additional arguments to pass to the `apply_fn`.
+        Each additional argument should be a list of length 2. The first
+        element of each list will be passed to the `apply_fn` when processing
+        pre-change date data; the second element will be used for the
+        post-change date data.
```

Comment:
it looks like we never actually call this with different arguments to be used before vs after the change date. was that intended?

Reply:
Yeah, the before/after change date data is always processed using the same args.

The alternative implementation is to pass single values as `*args` instead of lists, and duplicate them to the right length within `apply_thres_change_date`.